pkgs/development/mobile/androidenv/emulate-app.nix

```diff
@@ -1,24 +1,21 @@
 { composeAndroidPackages, stdenv }:
 { name, app ? null
-, platformVersion ? "16", abiVersion ? "armeabi-v7a", systemImageType ? "default", useGoogleAPIs ? false
```

Comment:
Does useGoogleAPIs not have an effect elsewhere?

Reply:
No, it's unused. The original author forgot to pass it to `composeAndroidPackages`.